itou/approvals/management/commands/import_pe_approvals.py

```diff
@@ -60,56 +60,54 @@ def handle(self, file_path, dry_run=False, **options):
         bulk_create_queue = []
         chunk_size = 5000
 
-        # https://openpyxl.readthedocs.io/en/latest/optimized.html#read-only-mode
-        wb = openpyxl.load_workbook(file_path, read_only=True)
-        ws = wb.active
+        df = pd.read_excel(file_path)
 
         self.stdout.write("Ready.")
         self.stdout.write("Creating approvals… 0%")
 
         last_progress = 0
-        for i, row in enumerate(ws.rows):
+        for i, row in df.iterrows():
 
             if i == 0:
                 # Skip XLSX header.
                 continue
 
-            if not row[0].value:
+            if not row[0]:
```

Comment:
Un peu obscur ce test que la première cellule est vide. Comme le suggère le commentaire ci-dessous, ce serait plus propre de tester qu'on est sur la dernière ligne :

```
if idx == df.index[-1]:
    assert not row[0]
    continue
```

Reply:
Test de la dernière ligne trouvé sur https://stackoverflow.com/questions/43968692/check-if-last-row-in-pandas-df-iterrows

Reply:
C'est nickel, merci !